sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2828,7 +2829,8 @@ object GpuOverrides {
       "Writing data",
       ExecChecks((TypeSig.commonCudfTypes +
         TypeSig.DECIMAL.withPsNote(TypeEnum.DECIMAL, "Only supported for Parquet") +
-        TypeSig.STRUCT.withPsNote(TypeEnum.STRUCT, "Only supported for Parquet")).nested(),
+        TypeSig.STRUCT.withPsNote(TypeEnum.STRUCT, "Only supported for Parquet") +
```

Comment:
nit: would be good to have an API that takes an array of `TypeEnum` so we have to duplicate the message `Only supported for Parquet`

Reply:
We can create a s follow-on issue for this as an improvement if you want

Reply:
ok, Thx @razajafri, Will create a follow-on issue for this